src/Widgets/PopoverWidget.vala

```diff
@@ -42,13 +42,8 @@ public class Power.Widgets.PopoverWidget : Gtk.Grid {
         var device_list_revealer = new Gtk.Revealer ();
         device_list_revealer.add (device_list);
 
-        var device_separator = new Gtk.Separator (Gtk.Orientation.HORIZONTAL) {
-            margin_top = 3,
-            margin_bottom = 3
-        };
```

Comment:
What's the reasoning behind removing these separators?

Reply:
The Gtk.Grid packed into Gtk.Button has its own edges, so the separator seems to be duplicated.